src/config/config.js

```diff
@@ -210,7 +210,7 @@ const Config = {
   ceramicIndex: env.REACT_APP_CERAMIC_INDEX,
   ceramicLiveIndex: env.REACT_APP_CERAMIC_LIVE_INDEX,
   ceramicBatchSize: (env.REACT_APP_CERAMIC_BATCH_SIZE || 5),
-  ceramicPollInterval: parseInt(env.REACT_APP_CERAMIC_POLL_INTERVAL || 3600),
+  ceramicPollInterval: parseInt(env.REACT_APP_CERAMIC_POLL_INTERVAL || 3600) * 1000,
```

Comment:
what this  *1000?

Reply:
this makes interval in ms (to be just passed to setTimeout as is)

Reply:
particularly it made me easy to setup sources, using simple checks like ternary operator here: https://github.com/GoodDollar/GoodDAPP/pull/4015/files#diff-d15ab847926eb88ba4b415629bd4e4739c413121193d64bd292ecfcb72c5117eR242